docs/src/main/asciidoc/ap4k.adoc

```diff
@@ -36,7 +36,7 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
 == Enable Kubernetes support
 
 Quarkus offers the ability to automatically generate Kubernetes resources based on sane defaults and user supplied configuration. The implementation that takes care
-of generating the actual Kubernetes resources is provided by https://github.com/ap4k/ap4k/[ap4k].
+of generating the actual Kubernetes resources is provided by https://github.com/dekorateio/dekorate/[dekorate].
```

Comment:
Should we rename the guide to something more generic (kubernetes-resources.adoc for instance or whatever name sounds good)?

Reply:
That makes a lot of sense. The only reason I didn't do it is because I couldn't find where the file is linked so I didn't what to break anything.

Reply:
I renamed it to `kubernetes-resources.adoc`. There aren't any references to `ap4k.adoc` here so I guess we're good

Reply:
Yeah, I need to remember to change it in the guides yaml file on the website when I release :).

Reply:
I'll try and remind you 😉